plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -1028,7 +1028,7 @@ public void testEnum()
     /**
      * @see #testTimeCoercion
      */
-    @Test(dataProvider = "testTimestampDataProvider")
+    @Test(dataProvider = "sessionZonesDataProvider")
```

Comment:
cc: @findepi on the data provider name change. The providers are now reused across methods with different names so making the provider name match the test method name isn't possible.

I believe this change is good.